server/graphQl/resolvers.js

```diff
@@ -92,17 +93,17 @@ const shortenUrl = async ({ userId, longUrl }) => {
     error.data = errors;
     throw error;
   }
-
-  // check if url already exists for the user in the db
-  const existingUrl = await Url.findOne({ longUrl: longUrl });
-  if (userId === 'guest') {
+  // If user is not logged in then assign guest userId
+  if (userId == null) {
```

Comment:
```suggestion
  if (userId === null) {
```
Always use `===` in JS.